src/components/forms/DisciplineForm/DisciplineCreateForm.scss

```diff
@@ -0,0 +1,80 @@
+@import 'src/scss/vars';
+
+.discipline-form {
+  width: 940px;
+
+  &__info-label {
+    font-weight: bold;
+    margin-bottom: 10px;
+  }
+
+  &__info {
+    &--correct {
```

Comment:
i see one nested block without basic styles, then it should be `&__info--correct{...`